src/devices/Card/CreditCard/category.txt

```diff
@@ -0,0 +1 @@
+Card
```

Comment:
lowercase, also you might want to add `rfid` here since it's related

Reply:
made the changes in the other files as well